.github/workflows/mfe-s3-bucket-deployment.yml

```diff
@@ -33,6 +33,15 @@ on:
         description: Deployment environment (prod, stage)
         type: string
         required: true
+      BUCKET_NAME:
+        type: string
+        required: false
+      PUBLIC_PATH_CDN:
+        type: string
+        required: false
+      AWS_CLOUDFRONT_DISTRIBUTION_ID:
+        type: string
+        required: false
```

Comment:
the bucket and aws_cloudfront are attached to https://github.com/nelc/actions-hub/pull/10/files#diff-c80649a9b4831cd14c8699c5fdbe0cb4bb85eea9e979ecb99158b1bdd3b26b95R130-R132

secrets, but in my opinion is ok to keep them in vars

Reply:
Sorry I don't understand your comment and the link redirects me to this comment